app/controllers/users_controller.rb

```diff
@@ -1,4 +1,37 @@
 class UsersController < ApplicationController
+
+  def show
+    @user = User.find_by id: params[:id]
+    unless @user
+      flash[:danger] = "NIL"
```

Comment:
em sửa lại dòng thông báo cho phù hợp hơn nhé, chẳng hạn như "User not found". Sau đó em viết I18n cho nó nữa nhé